docs/src/content/docs/en/spikes/tech-spikes-split-verify-of-buildkite.mdx

```diff
@@ -5,11 +5,89 @@ description: Split current token for BuildKite
 
+## Old Api
+- Api Design
+```
+paths: api/v1/pipelines/{pipelineType}
+method: post
+request: {
+  PipelineParam:{
+  token:"your_token",
+  startTime: "xxx",
+  endTime:"xxx",
+  } 
+}
+responses:{
+  "pipelineList": 
+  [{
+    "id": "xxx",
+    "name": "xxx",
+    "orgId": "xxx",
+    "orgName": "xxx",
+    "repository": "xxx",
+    "steps": [
+     "xxx",
+      ]
+    }]
+}
+```
+- Sequence Diagram
+```plantuml
+@startuml
+skin rose
+title - Heartbeat - Get BuildKite Info
+
+participant FrontEnd
+participant BuildKiteController
+participant BuildKiteService
+participant BuildKiteFeignClient
+participant RestResponseEntityExceptionHandler
+participant BuildKite
+
+' group Get BuildKite Info
+    FrontEnd -> BuildKiteController: request: PipelineParam{token,startTime,endTime}, type
+    activate BuildKiteController
+    BuildKiteController -> BuildKiteService: PipelineParam{token,startTime,endTime}
+    activate BuildKiteService
+    BuildKiteService -> BuildKiteFeignClient: [1] token
+    activate BuildKiteFeignClient
+    BuildKiteFeignClient -> BuildKite: [1] v2/access-token
+    BuildKite -> BuildKiteFeignClient: [1] 200 scopes[xxx,xxx,...]
+    BuildKiteFeignClient -> BuildKiteService: [1] 200 scopes[xxx,xxx,...]
+    BuildKiteService -> BuildKiteFeignClient: [2] token
+   
+    BuildKiteFeignClient -> BuildKite: [2] v2/organizations
+    BuildKite -> BuildKiteFeignClient: [2] BuildKiteOrganizationsInfoList{name,slug}
+    BuildKiteFeignClient -> BuildKiteService: [2] BuildKiteOrganizationsInfoList{name,slug}
+    BuildKiteService -> BuildKiteFeignClient: [3] token,slug,startTime,endTime,page,perPage
+    BuildKiteFeignClient -> BuildKite: [3] v2/organizations/{organizationId}/pipelines?page={page}&per_page={perPage}
+    BuildKite -> BuildKiteFeignClient: [3] BuildKitePipelineDTOList{id,url,name ...}
+alt Verification success
+    BuildKiteFeignClient --> BuildKiteService: BuildKitePipelineDTOList{id,url,name ...}
+    deactivate BuildKiteFeignClient
+    BuildKiteService --> BuildKiteController: BuildKiteResponseDTO{pipelineList}
+    deactivate BuildKiteService
+    BuildKiteController --> FrontEnd: response: BuildKiteResponseDTO{pipelineList}
+    deactivate BuildKiteController
+else Verification failed
+    BuildKiteFeignClient --> RestResponseEntityExceptionHandler: throw 401 UnauthorizedException
+    deactivate BuildKiteFeignClient
+    activate RestResponseEntityExceptionHandler
+    RestResponseEntityExceptionHandler -> FrontEnd: Failed to get BuildKite info_status: 401 UNAUTHORIZED, reason:xxx
+    deactivate RestResponseEntityExceptionHandler
+    ' deactivate BuildKiteService
+    ' deactivate BuildKiteController
+end
+@enduml
+```
+
 ## Solutions
 ### 1. Verify the BuildKite token
+The new api need to be called when the Verify button is clicked. If return 200, the verification is successful.Otherwise 401 exception will be thrown.
+
 - Api Design
 ```
-paths: api/v1/BuildKite/verify
+paths: api/v1/pipelines/verify
```

Comment:
type: "BuildKite" should be integrated with the path, not put it into body
This API design needs to be discussed again

Reply:
done